dev-lang/erlang/erlang-21.0.5.ebuild

```diff
@@ -0,0 +1,220 @@
+	http://erlang.org/download/otp_doc_man_${UPSTREAM_V}.tar.gz -> ${PN}_doc_man_${UPSTREAM_V}.tar.gz
+	doc? ( http://erlang.org/download/otp_doc_html_${UPSTREAM_V}.tar.gz -> ${PN}_doc_html_${UPSTREAM_V}.tar.gz )"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~x64-solaris"
+
+IUSE="doc emacs +hipe java +kpoll libressl odbc +pgo sctp ssl systemd tk +wxwidgets"
+
+RDEPEND="
+	sys-libs/ncurses:0
+	sys-libs/zlib
+	emacs? ( virtual/emacs )
+	java? ( virtual/jdk:* )
+	odbc? ( dev-db/unixODBC )
+	sctp? ( net-misc/lksctp-tools )
+	ssl? (
+		!libressl? ( >=dev-libs/openssl-0.9.7d:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+	systemd? ( sys-apps/systemd )
+	tk? ( dev-lang/tk:0 )
+	wxwidgets? ( x11-libs/wxGTK:${WX_GTK_VER}[X,opengl] )
+"
+DEPEND="${RDEPEND}
+	dev-lang/perl
+"
+
+S="${WORKDIR}/otp-OTP-${PV}"
+
+PATCHES=(
+		"${FILESDIR}/18.2.1-wx3.0.patch"
+		"${FILESDIR}/${PN}-20.3.2-dont-ignore-LDFLAGS.patch"
+		"${FILESDIR}/${PN}-add-epmd-pid-file-creation-for-openrc.patch"
+	)
+
+SITEFILE=50"${PN}"-gentoo.el
+
+# Taken from the upstream build script, build_otp
+erlang_create_lib_configure_in() {
+	local bootstrap_apps="erts lib/asn1 lib/compiler lib/hipelib/ic lib/kernel
+			 lib/parsetools lib/sasl lib/snmp lib/stdlib lib/syntax_tools"
+	local sdirs=
+	for lib_app in ${bootstrap_apps}; do
+		case "${lib_app}" in
+			lib/*)
+				if [[ -f "${lib_app}/configure.in" ]]; then
+					mv "${lib_app}/configure.in" "${lib_app}/configure.ac" || die
+					app=`echo "${lib_app}" | sed "s|lib/\(.*\)|\1|"`
+					sdirs="${sdirs}test ! -f ${app}/configure || AC_CONFIG_SUBDIRS(${app}/.)\n" || die
+				fi;;
+			*)
+			;;
+		esac
+	done
+
+	local sed_bootstrap="s%@BOOTSTRAP_CONFIGURE_APPS@%$sdirs%g"
+
+	sdirs=
+	for lib_app in lib/*; do
+		local is_bapp=false
+		for bapp in ${bootstrap_apps}; do
+			test "${bapp} != ${lib_app}" || { "${is_bapp}"=true; break; }
+		done
+		if [[ "${is_bapp}" = false ]] && [[ -f "${lib_app}/configure.in" ]]; then
+			mv "${lib_app}/configure.in" "${lib_app}/configure.ac" || die
+			app=`echo "${lib_app}" | sed "s|lib/\(.*\)|\1|"` || die
+			sdirs="${sdirs}    test ! -f ${app}/configure || AC_CONFIG_SUBDIRS(${app}/.)\n"
+		fi
+	done
+
+	local sed_non_bootstrap="s%@NON_BOOTSTRAP_CONFIGURE_APPS@%$sdirs%g"
+
+	rm -f lib/configure.in || die
+	sed "$sed_bootstrap;$sed_non_bootstrap" > lib/configure.ac < lib/configure.in.src || die "Failed to create lib/configure.ac"
+
+}
+
+# Taken from the upstream build script, build_otp
+erlang_distribute_config_helpers() {
+	local aclocal_dirs=". ./lib/erl_interface ./lib/odbc ./lib/wx ./lib/megaco"
+	local autoconf_aux_dirs="./lib/common_test/priv/auxdir ./lib/erl_interface/src/auxdir ./lib/common_test/test_server ./lib/wx/autoconf"
+
+	local aclocal_master="./erts/aclocal.m4"
+	local install_sh_master="./erts/autoconf/install-sh"
```

Comment:
Is there really a reason to call external install script when you have environment with guaranteed working `install` tool? But then, is there a point to play `install` when for our use case regular `cp` would work just fine?